src/export/option.ts

```diff
@@ -108,8 +108,11 @@ interface ToolboxComponentOption extends ToolboxOption {
     }
 }
 
+export { SliderDataZoomOption, InsideDataZoomOption };
 export type DataZoomComponentOption = SliderDataZoomOption | InsideDataZoomOption;
+export { ContinousVisualMapOption, PiecewiseVisualMapOption };
 export type VisualMapComponentOption = ContinousVisualMapOption | PiecewiseVisualMapOption;
+export { LegendOption, ScrollableLegendOption };
```

Comment:
`LengendOption` can be renamed to `PlainLegendComponentOption`

Reply:
✔️